pkg/controller/dual-pods/controller.go

```diff
@@ -36,6 +38,7 @@ import (
 )
 
 const ControllerName = "dual-pods-controller"
+const GPUMapName = "gpu-map"
```

Comment:
Please add a comment (and/or documentation) that specifies the expected format of gpu-map.